package.json

```diff
@@ -103,6 +104,7 @@
     "fs-extra": "^1.0.0",
     "fuse.js": "^2.2.0",
     "glob": "^7.0.0",
+    "glslify": "^4.0.0",
```

Comment:
I made a custom build with `npm run build` for testing purposes (http://codepen.io/monfera/pen/ObrRGP) and it worked fine, the transforms were added [here](https://github.com/plotly/plotly.js/pull/1256/files#diff-75cb9bb70bfe9c024a0191bc048593a6L41) and [here](https://github.com/plotly/plotly.js/pull/1256/files#diff-6965cd2e51c744999999f8eff1de2c3cL25). Please let me know if it's insufficient, and in this case, how should I go about testing whether the `package.json` change was successful or not, because the build seems good as it is.

Perhaps you're referring to users who just put `plotly.js` in as one of their dependencies in their `package.json` and install their system with `npm install`, instead of using a bundle? Should I test for this?